### PR TITLE
[WIP] Fix process document url

### PR DESCRIPTION
**Pull request by manycoding**
`process_document_url` was failing due to wrong parameter name. I also updated errors.

- [ ] Tests
- [ ] Docs

### CODE REVIEW

File: veryfi/__init__.py [hunk 1]
```diff
@@ -104,7 +105,7 @@ def _request(self, http_verb, endpoint_name, request_arguments, file_stream=None
                                          timeout=self.timeout)
 
         if response.status_code not in [200, 201, 202, 204]:
-            raise VeryfiClientError(response)
+            raise VeryfiClientError.from_response(response)
```
Comment (manycoding) [inline veryfi/__init__.py:108]: now raise returns neat error message like `BadRequest: 400, Bad or Missing parameters.`, not class str like `VeryfiClientError: <Response [400]>`

File: veryfi/errors.py [hunk 2]
```diff
@@ -1,22 +1,16 @@
-# -*- coding: utf-8 -*-
-from __future__ import absolute_import
-from __future__ import unicode_literals
-
-
 class VeryfiClientError(Exception):
-    required_fields = []
```
Comment (manycoding) [inline veryfi/errors.py:7]: I don't see where it's used, so I cleaned it.

File: veryfi/__init__.py (hunk 1 above)
Comment (dbirulia) [reply veryfi/__init__.py:108]: Good point.

File: veryfi/errors.py (hunk 2 above)
Comment (dbirulia) [reply veryfi/errors.py:7]: Great, thank you!

**Pull request merged**